cmd/satellite/main.go

```diff
@@ -229,6 +244,48 @@ func cmdNodeUsage(cmd *cobra.Command, args []string) (err error) {
 	return generateCSV(ctx, start, end, file)
 }
 
+func cmdValueAttribution(cmd *cobra.Command, args []string) (err error) {
+	ctx := process.Ctx(cmd)
+
+	// Parse the UUID
+	partnerID, err := uuid.Parse(args[0])
```

Comment:
What is main objective? to have report for all partners as one csv or one cvs for one partner? its not clear from ticket description. I think it make sense to have one report for all and then this parameter wont be needed.

Reply:
The design document says by partner ID.

Reply:
then you need to update `Use` field in `cobra.Command` and minimum args number.

Reply:
Got it. Missed that. Good catch.